README.md

```diff
@@ -26,21 +26,67 @@ There are two ways to run the tests:
 
 ## On Local Machine
 
-To run tests on your local machine, first you need to start the frontend server:
+### 1. Start the development server
 
-```
+To run tests on your local machine, first you need to start the development server:
+
+```bash
 yarn build && yarn preview
 ```
 
-Then you need to run the tests using
+### 2. Ensure Keys are in Local Keyring
+
+You need to ensure that the key `user1` is present in your local keyring. To find the mnemonic for this key, open the file `test/e2e/test.utils.js`. On `line 1` of this file, you will find an object named `mnemonics`, which contains the mnemonics for different keys, including `user1` and `user2`.
```

Comment:
minor:
We can remove "On `line 1` of this file" statement.
Could be a case when someone changes the file `test/e2e/test.utils.js` and `mnemonics` jumps to another line.

Reply:
Good point. Let me make the update.